### PR TITLE
feat(route): 增加浙江中医药大学教务处

### PR DESCRIPTION
**Pull request by i-CraftY**
## 该 PR 相关 Issue / Involved issue

Close #

## 完整路由地址 / Example for the proposed route(s)

```routes
/zcmu/jwc/1
/zcmu/jwc/2
/zcmu/jwc/3
/zcmu/jwc/4
/zcmu/jwc/5
```

## 新 RSS 检查列表 / New RSS Script Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [ ] 英文文档 EN
- [ ] 全文获取 fulltext
  - [ ] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [ ] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by CCraftY** [comment]
routes标识已修正 @NeverBehave

**Comment by NeverBehave** [comment]
LGTM

File: docs/university.md [hunk 1]
```diff
@@ -3247,6 +3247,18 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
 
 </Route>
 
+## 浙江中医药大学
+
+### 浙中医大教务处
```
Comment (TonyRL) [inline docs/university.md:3252]:
```suggestion
### 教务处
```

File: docs/university.md [hunk 2]
```diff
@@ -3247,6 +3247,18 @@ jsjxy.hbut.edu.cn 证书链不全，自建 RSSHub 可设置环境变量 NODE_TLS
...
+
+<Route author="CCraftY" example="zcmu/jwc/1" path="zcmu/jwc/:type?" :paramsDesc="['通知模块id']">
```
Comment (TonyRL) [inline docs/university.md:3254]:
```suggestion
<Route author="CCraftY" example="/zcmu/jwc/1" path="/zcmu/jwc/:type?" :paramsDesc="['通知模块id']">
```

File: lib/v2/zcmu/radar.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    'zcmu.edu.cn': {
+        _name: '浙江中医药大学',
+        '.': [
+            {
+                title: '浙江中医药大学',
```
Comment (TonyRL) [inline lib/v2/zcmu/radar.js:6]:
```suggestion
                title: '教务处',
```

**Comment by CCraftY** [comment]
updated

**Review by TonyRL — CHANGES_REQUESTED** [review]
Every item has its `pubDate`. Please add it as well.

**Pull request merged**